internal/cli/command_factory.go

```diff
@@ -153,7 +142,8 @@ func (factory *CommandFactory) Close() {
 // Run executes the command
 func (factory *CommandFactory) Run(cmd *cobra.Command) {
 	if err := cmd.Execute(); err != nil {
-		if _, ok := err.(errSupressUsage); !ok {
+		cause, isUsagePrinter := errors.Unwrap(err).(UsagePrinter)
```

Comment:
[nit] it's more idiomatic to use `ok` here instead of the `isUsagePrinter`

Reply:
👍 ... I generally go with these names when the `var` isn't confined to a very tight closure, so to avoid any potential
 (now or future) downstream shadowing of `ok`.  but i'll change it here (there's no shadowing, little chance this function/closure grows much more)